govc/session/ls.go

```diff
@@ -51,7 +52,8 @@ func (cmd *ls) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.OutputFlag, ctx = flags.NewOutputFlag(ctx)
 	cmd.OutputFlag.Register(ctx, f)
 
-	f.BoolVar(&cmd.r, "r", false, "Include current REST session (if any)")
+	f.BoolVar(&cmd.r, "r", false, "Include cached REST session (if any)")
+	f.BoolVar(&cmd.s, "S", false, "Include current SOAP session only")
```

Comment:
Yes, they can be used together. Could reword that, "Limit SOAP session list to current only" ?  It's a bit awkward since the REST API doesn't have a way to list all sessions.

Reply:
What if we simply drop `only` from that description?